src/main/resources/page/js/dummy-endpoint-caller.js

```diff
@@ -0,0 +1,8 @@
+class APICaller {
```

Comment:
What kind of APICaller?

Reply:
What do you think about RESTApiCaller?

Reply:
Is this is a generic solution, it can work, if it is a Dummy related API caller, then DummyRESTAPICaller would be a better choice.